app/workers/fetch_graph_worker.rb

```diff
@@ -48,12 +48,12 @@ def perform(pid, user_key)
     ActiveFedora::SolrService.commit
   end
 
-  def fetch_failed_callback(user, val)
-    Hyrax.config.callback.run(:ld_fetch_failure, user, val.rdf_subject.value)
-  end
+  # def fetch_failed_callback(user, val)
```

Comment:
We can remove this method as well. Or maybe add a comment to explain the reason for commenting out the method.

Reply:
These commented pieces of code will need to be reset. Its on my agenda for #986

Reply:
sounds good